### ISSUE TITLE
Vue dev tools does not show the state with pinia 0.2.0

### ISSUE THREAD

**Issue by dyamon-cz** [posva/pinia#394]
Hello, I have tried today a new version of Pinia 0.2.0 for Vue 2.

I have noticed an issue with Vue Devtools, they stopped showing state of pinia store, mutations are being displayed however state is not.

_(Vue Devtools recently changed default settings for loading state to manual "Load State" button, but that should not be related to this issue)_ 

I have setup reproduction in my repository.

Firstly after pulling a repo try to serve a master branch which has old pinia 0.1.0 which works correctly and displaying the state. (there is a clicking button in UI which increments counter) 
```bash
git clone https://github.com/dyamon-cz/pinia-bug-store-state.git
cd pinia-bug-store-state
npm ci #must use ci to get data from package-lock.json and do not update packages
npm run serve
```
After clicking on the "click" button in the UI the state is being properly displayed.

![image](https://user-images.githubusercontent.com/13029312/111760804-1adef700-88da-11eb-8162-346d5811f9e5.png)

Then switch to bug-devtools branch reinstall node modules and serve. This branch has pinia 0.2.0 set up in main.js based on documentation.

```bash
git checkout --track origin/bug-devtools
npm ci #must use ci to get data from package-lock.json and do not update packages
npm run serve
```

Now there is no state displayed after clicking on the button.

![image](https://user-images.githubusercontent.com/13029312/111761386-cbe59180-88da-11eb-9bac-7f6582f733f0.png)

Thank you very much for help!

**Comment by posva** [comment]
Have you unchecked the new backend for veux in Vue devtools settings?
pinia doesn't work with it (#19)

**Comment by posva** [comment]
It seems like something broke the devtools integration...
If anybody wants to contribute, looking at the devtools.ts in the v1 branch and https://github.com/posva/pinia/issues/19#issuecomment-792831188 should help to find a solution. There must be some events that are not being listened/emitted correctly

**Comment by dyamon-cz** [comment]
Yes vuex is not used, I'll take a look at devtools.ts thanks for pointing out the right direction.

**Comment by posva** [comment]
Closed in 043ae2a
